docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,442 @@
+# or
+datum convert -if voc -i <path/to/voc> -f coco -o <path/to/output/dir>
+```
+
+Also it is possible using filters for converting, check
+[user manual](../docs/user_manual.md#filter-project)
+for information about filtering:
+
+``` bash
+datum convert -if voc -i <path/to/voc> \
+    -f yolo -o <path/to/output/dir> \
+    --filter-mode FILTER_MODE \
+    --filter '<xpath filter expression>'
+```
+
+Some formats have extra arguments for converting, run `datum export -f FORMAT -- -h`
+for more information. For example voc_segmentation format has extra argument
+`--label_map` which get opportunity to load user color map for segmentation mask:
+
+``` bash
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+```
+
+Also you can use original voc format of label map, for example:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Export to Pascal VOC
+
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset,
+by default datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
+
+``` bash
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+```
+
+Or you can use original Pascal VOC label map:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Datumaro functionality
+
+Datumaro supports filtering, transformation, merging and etc. for all formats
+and for the Pascal VOC format in particular. Follow
+[user manual](../docs/user_manual.md)
+to get more information about these operations.
+
+There are few examples of using datumaro operations to solve
+particular problems:
+
+- We can convert Pascal VOC dataset to Market-1501 format. But Market-1501 dataset
+only has a `person` class, marked with a bounding box. And to perform the conversion
+we could filter the Pascal VOC dataset. With datumaro we can do it like this
+
+``` bash
+# create datumaro project with Pascal VOC dataset
+datum import -o <path/to/voc/dataset> --name project
+
+# convert labeled shapes into bboxes
+datum transform -t shapes_into_boxes
+cd project-shapes_to_boxes
+
+# keep only person class items
+datum filter --mode items+annotations \
+    -e '/item[label="person"]' \
+    -o <path/to/output/project>
+
+# delete other labels from dataset
+datum trasnform -t remap_labels -- -l person:person --default delete \
```

Comment:
I suggest to provide a concrete working example instead of a list of unrelated commands. Ensure they can be executed.

Reply:
But, this example solve concrete task: preparing Pascal VOC dataset for converting to other dataset format, using related set of Datumaro operations. Isn`t it?

Reply:
This example cannot be executed and it wouldn't create the expected dataset. Please make sure these commands do what they are supposed to do.